spec/unit/identity_spec.rb

```diff
@@ -47,4 +47,27 @@ def id
       end
     end
   end
+
+  describe '#to_key' do
```

Comment:
Just fixed a missing "#" here so it'll probably take a second run through Travis. Sorry! :)